ReactAndroid/src/main/third-party/android/support/v7/appcompat-orig/BUCK

```diff
@@ -38,13 +38,13 @@ prebuilt_jar(
 
 genrule(
     name = 'classes-unpacker-cmd',
-    cmd = '$(exe :aar-unpacker) $(location :appcompat-binary-aar) "classes.jar" $OUT',
```

Comment:
These quotes broke the script on Windows for some reason they appeared in the path inside the script (something like /path/path/"classes.jar"). Still works properly without them on both windows and mac.